SU-SWS/ansible-playbooks
# SITES-929: Scale up

# READY FOR REVIEW

- See https://github.com/SU-SWS/acsf-cardinald7/pull/105
- See https://github.com/SU-SWS/acsf-leland/pull/19
- **Requires** https://github.com/SU-SWS/ansible-sync/pull/3
- Includes work from #65 

# Summary
- Go faster

# Needed By (Date)
- 8:30PM Tuesday, Oct. 9.

# Criticality
- How critical is this PR on a 1-10 scale? 10/10
- We can't do the people.stanford.edu cutover without it

# Steps to Test

1. Check out this branch
2. Check out the `SITES-929` branch of `ansible-sync`
3. Copy the `acsf.aliases.drushrc.php` file from `ansible-sync` into `roles/setup-local/templates/`
4. Copy several hundred sites to `dev` or `test`
5. Feel the wind in your hair

# Affected Projects or Products
- **ARRYTHING!**

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-929
- Other PRs (see above)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
